platform/security/sts/security-sts-attributequeryclaimshandler-common/src/test/java/org/codice/ddf/security/claims/attributequery/common/TestAttributeQueryClient.java

```diff
@@ -100,13 +63,11 @@ protected HttpsURLConnection createHttpsUrlConnection(URL url) {
 
     private static final String ISSUER = "testIssuer";
 
-    private String username = "admin";
+    private static final String EXTERNAL_ATTRIBUTE_STORE = "https://localhost:8993/";
```

Comment:
Is this going to break when this unit tests runs when a DDF is also running on the system?